### ISSUE TITLE
Tests for new `cellpose_segmentation` parameters

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-tasks-core#208]
With #207 we are adding two new parameters:
* `pretrained_model`, see #60
* `label_name`, see #206 

Those will be available in version `0.3.4`. Let's make sure to include these features in some examples, to make sure that they are working as expected (it's somewhat cumbersome to test them in the CI, especially for the custom model).

**Comment by jluethi** [comment]
Oh, awesome! Once 0.3.4 is released, I'll port one of the organoid examples to the v1.x version, because we have relevant pre-trained networks for those. Will also test the custom label_name for them there.

**Comment by jluethi** [comment]
I moved the GPU usage discussion to a separate issue here: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/215

A bit of context on the different cellpose models:
The main `Cellpose` class is a wrapper for both the `CellposeModel` and the `SizeModel`. We just need the CellposeModel. The SizeModel estimates reasonable diameters (and thus the wrapper `Cellpose` class returns both the mask, flows, styles from the cellpose model, as well as the diams result from `SizeModel`).

I'm testing the modified version with `CellposeModel` now and modifying the output handling to see if we can easily switch to the core `CellposeModel` class, away from the wrapper class :)

**Comment by tcompa** [comment]
> I'm testing the modified version with `CellposeModel` now and modifying the output handling to see if we can easily switch to the core `CellposeModel` class, away from the wrapper class :)

That would be interesting, since (if I remembered correctly) the `flows` part was quite heavy in terms of memory. If it is useful, it does work at the moment. If it is not useful, let's drop it.

**Comment by jluethi** [comment]
The `label_name` part works as intended 🎉
And I'll submit a PR to switch to `CellposeModel` after a few more tests. It looks like that model works well and can take the pretrained model.
Regarding `flows`, those are part of the `CellposeModel` calculations and we'll need that part. I doubt we'd be getting major memory gains here.

**Comment by tcompa** [comment]
Great, thanks.
There is already a `cellpose_fixes` branch available, linked to PR #218. Feel free to push there, if it's easier.

**Comment by jluethi** [comment]
Ok, added the changes to the #218 PR now: https://github.com/fractal-analytics-platform/fractal-tasks-core/pull/218/commits/a3e6e4bba64fbc9f1523c11655f814ff0231e22e

Using the hummingbird model from Maurice from the Liberali lab, this can create better organoid segmentation, see here:

![Screenshot 2022-11-21 at 17 35 59](https://user-images.githubusercontent.com/18033446/203110629-1ae2db62-04d9-435b-8755-f4b1d8d6c1e7.png)

(it's not perfect yet, one organoid with sparser cells gets missed. But it stops cutting organoids apart. Parameters could be tweaked a bit further to the hopefully get the optimal performance)

**Comment by jluethi** [comment]
Here's an example using the new parameters: https://github.com/fractal-analytics-platform/fractal-demos/issues/20

We can close this issue when the PR gets merged, both things will then work as intended.
